src/main/java/io/jenkins/plugins/orka/client/DeploymentRequest.java

```diff
@@ -14,8 +14,17 @@
     @SuppressFBWarnings("URF_UNREAD_FIELD")
     private String node;
 
+    @SuppressFBWarnings("URF_UNREAD_FIELD")
+    private String scheduler;
+
     public DeploymentRequest(String vmName, String node) {
         this.vmName = vmName;
```

Comment:
Samo about this ctor

Reply:
Same about the ctor

Reply:
The less specific one needs to call the more specific one. Currently, it is the other way around